src/NuGet.Clients/VsConsole/Console/WpfConsole/WpfConsole.cs

```diff
@@ -31,6 +31,10 @@ internal interface IPrivateWpfConsole : IWpfConsole
         "Microsoft.Maintainability",
         "CA1506:AvoidExcessiveClassCoupling",
         Justification = "We don't have resources to refactor this class.")]
+    [SuppressMessage(
+        "Microsoft.VisualStudio.Threading.Analyzers",
+        "VSTHRD010",
+        Justification = "We don't have resources to refactor this class.")]
```

Comment:
Fixing VSTHRD010 issues shouldn't usually require refactoring a class. It's a matter of adding some ThrowIfNotOnUIThread calls to the top of some of your methods, and occasionally fixing the caller to switch threads first. So would using the same NuGet/Home#4833 justification instead of this refactor one make more sense?